presto-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,130 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'presto-alerts',
+        rules: [
+          {
+            alert: 'PrestoHighInsufficientFailures',
+            expr: |||
+              increase(com_facebook_presto_execution_QueryManager_InsufficientResourcesFailures_TotalCount[5m]) > %(alertsHighInsufficientResourceErrors)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'The amount of failures that are occurring due to insufficient resources are scaling, causing saturation in the system.',
+              description:
+                (
+                  'The number of insufficient resource failures on {{$labels.instance}} is {{ printf "%%.0f" $value }} which is greater than the threshold of %(alertsHighInsufficientResourceErrors)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'PrestoHighTaskFailuresWarning',
+            expr: |||
+              increase(com_facebook_presto_execution_TaskManager_FailedTasks_TotalCount[5m]) > %(alertsHighTaskFailuresWarning)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'The amount of tasks that are failing is increasing, this might affect query processing and could result in incomplete or incorrect results.',
+              description:
+                (
+                  'The number of task failures on {{$labels.instance}} is {{ printf "%%.0f" $value }} which is above the threshold of %(alertsHighTaskFailuresWarning)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'PrestoHighTaskFailuresCritical',
+            expr: |||
+              increase(com_facebook_presto_execution_TaskManager_FailedTasks_TotalCount[5m]) / clamp_min(increase(com_facebook_presto_execution_TaskManager_FailedTasks_TotalCount[10m]), 1) * 100 > %(alertsHighTaskFailuresCritical)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'The amount of tasks that are failing has reached a critical level. This might affect query processing and could result in incomplete or incorrect results.',
+              description:
+                (
+                  'The number of task failures on {{$labels.instance}} is {{ printf "%%.0f" $value }} which is above the threshold of %(alertsHighTaskFailuresCritical)s%%s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'PrestoHighQueuedTaskCount',
+            expr: |||
+              increase(com_facebook_presto_execution_QueryManager_QueuedTaskCount[5m]) > %(alertsHighQueuedTaskCount)s
```

Comment:
going as far back as 7 days, no data showing up for this query (no threshold). have you been able to get load on this metric? Looks like a pretty simple query, so either you just haven't been able to get load on it or is there a typo in the metric name?